### DISCUSSION TITLE
Add option for loop voltage fudge factor

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2884] [Ideas]
In GitLab by @ajpearcey on Jun 19, 2023, 11:29

To help the loop voltage calculate agree with other transport code we should add in a user set coefficient.

**Comment by mkovari** [comment]
Hazardous but probably necessary.

**Comment by jmorris-uk** [comment]
There is a plasma resistivity factor in the code that was used to match PLASMOD in the past. Would probably be worth reviewing that in combination with the loop voltage.

**Comment by mkovari** [comment]
We shouldn't be adding in fudge factors without warning the users.  

It would be good to have a better physics-based 0D formula for plasma resistivity if we can find one.  The formula we are using is from the 1980s.
